pkg/common/config/config.go

```diff
@@ -376,6 +380,9 @@ func validateConfig(ctx context.Context, cfg *Config) error {
 		if !insecure {
 			vcConfig.InsecureFlag = cfg.Global.InsecureFlag
 		}
+		if setCfgGlobalvCenter {
+			cfg.Global.VCenterIP = vcServer
```

Comment:
Can we check if it is not already set and then assign the value

Reply:
added check

```
		if setCfgGlobalvCenter && cfg.Global.VCenterIP == ""{
			cfg.Global.VCenterIP = vcServer
		}
```